src/main/java/com/teamabnormals/abnormals_core/core/mixin/client/PlayerRendererMixin.java

```diff
@@ -0,0 +1,37 @@
+package com.teamabnormals.abnormals_core.core.mixin.client;
+
+import com.mojang.blaze3d.matrix.MatrixStack;
+import com.teamabnormals.abnormals_core.client.RewardHandler;
+import com.teamabnormals.abnormals_core.common.world.storage.tracking.IDataManager;
+import net.minecraft.client.entity.player.AbstractClientPlayerEntity;
+import net.minecraft.client.renderer.IRenderTypeBuffer;
+import net.minecraft.client.renderer.entity.PlayerRenderer;
+import net.minecraft.util.text.ITextComponent;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.UUID;
+
+@Mixin(PlayerRenderer.class)
+public final class PlayerRendererMixin {
+
```

Comment:
This blank line should be removed or add another blank line after the method. It's more consistent with the styling in the other code in this repository.